Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.12-slim
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+RUN mkdir -p /files/media

```

Comment:
Consider combining this `RUN` command with the previous `RUN` command to reduce the number of layers in the Docker image. This can be done by chaining commands with `&&`.